Documentation/cmdref/cilium-agent.md

```diff
@@ -229,6 +229,7 @@ cilium-agent [flags]
       --log-system-load                                         Enable periodic logging of system load
       --mesh-auth-monitor-queue-size int                        Queue size for the auth monitor (default 1024)
       --mesh-auth-mtls-listener-port int                        Port on which the Cilium Agent will perfom mTLS handshakes between other Agents
+      --mesh-auth-rotated-queue-size int                        The size of the queue for signaling rotated identities. (default 1024)
```

Comment:
shouldn't we name this `--mesh-auth-rotated-identities-queue-size` as mentioned in the comment -> with "identities"?